### ISSUE TITLE
error: implicit instantiation of undefined template

### ISSUE THREAD

**Issue by yurivict** [s-will/LocARNA#58]
clang8 on FreeBSD 12 complains:
```
In file included from LocARNA/alphabet.hh:87:
LocARNA/alphabet.icc:27:22: error: implicit instantiation of undefined template 'std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >'
        std::copy_n(s.begin(), N, this->begin());
                     ^
/usr/include/c++/v1/iosfwd:210:32: note: template is declared here
    class _LIBCPP_TEMPLATE_VIS basic_string;
                               ^
```

**Comment by s-will** [comment]
Would this patch fix your problem?
diff --git a/src/LocARNA/alphabet.icc b/src/LocARNA/alphabet.icc
index 83e6b43..30f8984 100644
--- a/src/LocARNA/alphabet.icc
+++ b/src/LocARNA/alphabet.icc
@@ -1,4 +1,5 @@
 #include <algorithm>
+#include <string>
 
 namespace LocARNA {

This is in the devel branch but not in the RC7 already.

**Comment by s-will** [comment]
I assume this is fixed in v2.0.0RC8
